chalice/app.py

```diff
@@ -205,6 +209,21 @@ def serve_swagger_ui():
                                 headers={"Content-Type": "text/html"},
                                 body=swagger_ui_html)
 
+    @app.route("/version")
```

Comment:
No, I think we should not update the Swagger service definition with this route.

1. This route is diagnostic in nature and not part of the main service API. It's intended primarily for the service operators.
2. Including and documenting this route would involve multiple levels of awkwardness (it would have to be moved out of this location and into a connexion routed location, relocated under the /v1/ prefix, and it would show up in the CLI and bindings, mixing it in with the main API, which I think is undesirable).

Reply:
Fair enough.